__tests__/frontmatter.js

```diff
@@ -123,3 +124,18 @@ function validateGlossaryFrontmatter({ frontmatter }) {
 		expect(frontmatter).toHaveProperty(requiredProp)
 	})
 }
+
+function validateAriaProperties(accessibilityRequirements) {
+	const aria12Key = Object.keys(accessibilityRequirements).find(key => key.startsWith('aria12:'))
```

Comment:
At first was thinking to search for `ariaxx` but if the number ever changes many more places will need to update this anyway

Reply:
What would you say about flipping this around and requiring a title for everything except WCAG criteria and techniques? That would be anything that doesn't start with either `/wcag-technique:/` or `/wcag2\d:/`